src/app/routes/article/handleEmptyParagraphBlocks/index.js

```diff
@@ -0,0 +1,27 @@
+import path from 'ramda/src/path';
+import assocPath from 'ramda/src/assocPath';
+
+const getBlocks = path(['content', 'model', 'blocks']);
+const setBlocks = assocPath(['content', 'model', 'blocks']);
+const setTextBlock = assocPath(['model', 'blocks']);
+
+const isEmptyParagraph = block => {
+  return block.type === 'paragraph' && !block.model.text;
```

Comment:
Is model always defined?
```suggestion
  return block.type === 'paragraph' && !block.model?.text;
```

Reply:
I think model and model.blocks will always be defined: https://www.test.bbc.com/arabic/articles/cl5m3453w36o.json
Paragraph contains a fragment block, even when empty